SU2_CFD/include/variables/CDiscAdjVariable.hpp

```diff
@@ -75,40 +76,69 @@ class CDiscAdjVariable final : public CVariable {
 
   /*!
    * \brief Get the Sensitivity at the node
+   * \param[in] iPoint - Point index
    * \param[in] iDim - spacial component
    * \return value of the Sensitivity
    */
   inline su2double GetSensitivity(unsigned long iPoint, unsigned long iDim) const override {
     return Sensitivity(iPoint,iDim);
   }
 
-  // TK:: Description missing
+  /*!
+   * \brief Set/store the dual time contributions to the adjoint variable. Contains contributions from 2 timesteps for dual time 2nd order.
+   * \param[in] iPoint - Point index
+   * \param[in] iVar - Index of the variable
+   * \param[in] der - Derivative value
+   */
   inline void SetDual_Time_Derivative(unsigned long iPoint, unsigned long iVar, su2double der) override {
     DualTime_Derivative(iPoint,iVar) = der;
   }
 
-  // TK:: Description missing
+  /*!
+   * \brief Set/store the dual time contributions to the adjoint variable for upcoming timestep.
+   * \param[in] iPoint - Point index
+   * \param[in] iVar - Index of the variable
+   * \param[in] der - Derivative value
+   */
   inline void SetDual_Time_Derivative_n(unsigned long iPoint, unsigned long iVar, su2double der) override {
     DualTime_Derivative_n(iPoint,iVar) = der;
   }
 
-  // TK:: Description missing
+  /*!
+   * \brief Return the dual time contributions to the adjoint variable. Contains contributions from 2 timesteps for dual time 2nd order.
+   * \param[in] iPoint - Point index
+   * \param[in] iVar - Index of the variable
+   * \return Derivative value
```

Comment:
Kudos for the briefs.
The params and returns on the other hand... I think they are superfluous, they just make you scroll more...
Moreover they are repeated a bunch of times, hence should be avoided.
"The only documentation that stays up to date is the one the compiler understands"

Reply:
I guess you're right. I'll remove the superfluous stuff